tests/garage/torch/algos/test_loss_function_utils.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import pytest
+import tensorflow as tf
+import torch
+
+import garage.tf.misc.tensor_utils as tf_utils
+import garage.torch.algos.loss_function_utils as torch_loss_utils
+from tests.fixtures import TfGraphTestCase
+
+
+class TestLossFunctionUtils(TfGraphTestCase):
+    # yapf: disable
+    @pytest.mark.parametrize('gae_lambda, rewards_val, baselines_val', [
+        (0.4,
+         np.ones((4, 6)),
+         np.zeros((4, 6))),
+        (0.8,
+         np.repeat(np.expand_dims([3, 1, 4, 1, 5, 9], axis=0), repeats=4, axis=0),  # noqa: E501
+         np.repeat(np.expand_dims(np.arange(6), axis=0), repeats=4, axis=0)),
+        (1.2,
+         np.ones((4, 6)),
+         np.repeat(np.expand_dims([1, 1, 2, 3, 5, 8], axis=0), repeats=4, axis=0)),  # noqa: E501
```

Comment:
this is very difficult to read and runs afoul of the style checker.

can you make these cases constants at the top of the file and reference them in the parameterize case?

it may actually be simpler and easier to read to just include them as large static numpy arrays at the top of the file than constructing them with `np.repeat`/`np.expand_dims`. As written, it would take a couple minutes to interpret what the actual inputs are here.

Reply:
fixed!